### PR TITLE
Authz and oc configuation fix

### PR DESCRIPTION
**Pull request by mojiiba**
This pull adds the authz config and fixes assumption related to oc config (the oc config should be a josn file instead of gnmi set in prototext format)

### CODE REVIEW

**Comment by mojiiba** [comment]
@gmacf could you please review this pr?

**Comment by gmacf** [comment]
> @gmacf could you please review this pr?

Taking a look now. Feel free to ping me if you'd like a review earlier as I don't get notifications for new PRs.

File: server/entitymanager/entitymanager.go [hunk 1]
```diff
@@ -103,13 +104,46 @@ func readOCConfig(path string) ([]byte, error) {
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Error opening file %s: %v", path, err)
 	}
-	req := &gpb.SetRequest{}
-	if err := prototext.Unmarshal(data, req); err != nil {
-		return nil, status.Errorf(codes.Internal, "File %s config is not a valid gnmi Setrequest: %v", path, err)
+	var v any
+	err1 := json.Unmarshal(data, &v)
+	if err1 != nil {
```
Comment (gmacf) [inline server/entitymanager/entitymanager.go:109]: Could you re-use 'err' here instead of making a new 'err1'?

**Review by gmacf — APPROVED** [review]

File: server/entitymanager/entitymanager.go [hunk 2]
```diff
@@ -39,7 +40,7 @@ import (
 	log "github.com/golang/glog"
 	bpb "github.com/openconfig/bootz/proto/bootz"
 	epb "github.com/openconfig/bootz/server/entitymanager/proto/entity"
-	gpb "github.com/openconfig/gnmi/proto/gnmi"
+	authzpb "github.com/openconfig/gnsi/authz"
```
Comment (gmacf) [inline server/entitymanager/entitymanager.go:43]: Could you make this "apb" instead? Our internal Go style guide recommends this.

**Comment by gmacf** [comment]
I resolved some conflicts from recent PRs and pushed them into this PR. It looks like there's still a couple areas I missed (mostly around naming or protos e.g. epb vs entity).

**Pull request merged**